src/router.js

```diff
@@ -260,14 +253,14 @@ const routes = (
                 />
                 <Route
                     path=":modelType/:modelId/sections"
-                    component={LoadableComponent({loader: () => import('./EditModel/EditDataSetSections.component')})}
+                    component={LoadableWithPreloadedStore({loader: () => import('./EditModel/EditDataSetSections.component')})}
                     onEnter={loadObject}
                     hideSidebar
                     disableTabs
                 />
                 <Route
                     path=":modelType/:modelId/dataEntryForm"
-                    component={LoadableComponent({loader: () => import('./EditModel/EditDataEntryForm.component')})}
+                    component={LoadableWithPreloadedStore({loader: () => import('./EditModel/EditDataEntryForm.component')})}
```

Comment:
To make it a bit more generic we could pass in the preloaded store to the component in addition to the loader function here, and use that in on L#37 in `LoadableComponent.js`.

Reply:
Can make it optional, and use the modelToEditStore default. Will be a bit verbose otherwise, imo.

Reply:
Yeah, but now the name should be: `LoadableWithPreloadedModelToEditStore` which is also a bit verbose. Doesn't matter a great deal so merge when you are ready.